portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,31 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
- */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function showInfo(number) {
+  let elementId = "info-" + number;
+  let infoDiv = document.getElementById(elementId);
+  let buttonDiv = document.getElementById("div-" + number);
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
-
-  // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  infoDiv.style.display = "block";
+  buttonDiv.style.display = "none";
 }
+
+window.onload = function() {
```

Comment:
If you follow up on my comment using `ul`, you could use this code to achieve the same behavior:
```javascript
const lists = document.querySelectorAll("div.info-container ul");
lists.forEach(function(element) {
  element.addEventListener("click", function() {
    const currentDisplay = this.style.display;
    this.style.display = currentDisplay !== "block" ? "block" : "none";
  }
});
```

Most likely the best solution would be to define a class to hide the elements, like this:
https://jsfiddle.net/gsd7ar0m/94/